docs/backend/behaviors.md

```diff
@@ -11,12 +11,305 @@ myst:
+
+| short name  | Title            | Desription                    |
+|-------------|------------------|-------------------------------|
+| plone.allowdiscussion  | Allow discussion | Allow discussion on this item |
+| plone.basic  | Basic metadata | Adds title and description fields. |
+| plone.categorization  | Categorization | Adds keywords and language fields. |
+| plone.collection  | Collection | Adds collection behavior |
+| plone.publication  | Date range | Adds effective date and expiration date fields. |
+| plone.dublincore | Dublin Core metadata | Adds standard metadata fields (equals Basic metadata +    Categorization + Effective range + Ownership  |
+| plone.eventattendees  | Event Attendees | Attendees extension for Events. |
+| plone.eventbasic  | Event Basic | Basic Event schema. |
+| plone.eventcontact  | Event Contact | Contact extension for Events. |
+| plone.eventlocation  | Event Location | Location extension for Events. |
+| plone.eventrecurrence  | Event Recurrence | Recurrence extension for Events. |
+| plone.excludefromnavigation  | Exclude From navigation | Allow items to be excluded from navigation |
+| plone.constraintypes| Folder Addable Constrains | Restrict the content types that can be added to folderish   content |
+| plone. textindexer | Full-Text Indexing | Enables the enhanced full-text indexing for a content type. If a field in the schema is marked with the `searchable` directive, its content gets added to the    `SearchableText` index in the catalog |
+| plone.leadimage  | Lead Image | Adds image and image caption fields |
+| plone.locking  | Locking | Locking support for dexterity |
+| plone.translatable  | Multilingual Support | Make this content type multilingual aware. Multilingual support must be installed. |
+| plone.namefromfilename | Name from file name | Automatically generate short URL name for content based on its    primary field file name
+| plone.namefromtitle | Name from title | Automatically generate short URL name for content based on its initial    title
+| plone.navigationroot  | Navigation root | Make all items of this type a navigation root |
+| plone.nextpreviousenabled  | Next previous navigation | Enable next previous navigation for all items of this type |
+| plone.nextprevioustoggle  | Next previous navigation toggle | Allow items to have next previous navigation enabled |
+| plone.ownership  | Ownership | Adds creator, contributor, and rights fields. |
+| plone.relateditems  | Related items | Adds the ability to assign related items |
+| plone.richtext  | RichText | Adds richtext behavior |
+| plone.shortname  | Short name | Gives the ability to rename an item from its edit form. |
+| plone.tableofcontents  | Table of contents | Adds a table of contents |
+| plone.thumb_icon | Thumbs and icon handling | Options to suppress thumbs and/or icons and to override thumb   size in listings, tables etc.
+| plone.versioning  | Versioning | Versioning support with CMFEditions |
```

Comment:
We could probably write at least a paragraph about each of these explaining what it does in more detail. But this is fine for a start. And maybe the detailed explanation of standard/core behaviors belongs elsewhere than the overview of how to use behaviors.

Reply:
Yes, but +1 to let get this in PR here in first and then start to enhance.

Reply:
> And maybe the detailed explanation of standard/core behaviors belongs elsewhere than the overview of how to use behaviors.

Indeed it would. https://github.com/plone/documentation/issues/1380

Reply:
Note that the table is no longer sorted. It was originally sorted by its Title, because that is what is displayed to the user. I would prefer that sort order be restored, unless there is a good reason not to do that.

Also please remove double-spaces in the table, as that shall appear to be intentional to the next author and copy-pasta may creep in.

Reply:
naturally I would expect it to be sorted by behavior name, at least as developer I would be confused if it is sorted by title.

Reply:
We should define who is the audience. Is it an end user, a developer, or another role?

Here is what the end user sees, sorted by `Title`, with no `short name` displayed:

https://6-classic.demo.plone.org/dexterity-types/Document/@@behaviors

If a developer, where would a developer find the list sorted by `short name`?

I'm fine with whatever makes the most sense for the intended audience.

Reply:
An end user would see the same without the names in the UI. So I would say this here is is more for Devs as reference.

Reply:
That's OK. We need to tell the developer where to find the list. My concern as a docs author is not being able to keep this list up to date because I have no clue where these short names are maintained. The only place I know where to look at this moment is for the end user, not developer.

Reply:
That still does not answer my question. How do you know the short names? Where do they come from? Is it in source code only? At the very least, give a link as reference for future PRs. Else I would say get rid of `short names` from the table altogether and refer only to what anyone can find here: https://6-classic.demo.plone.org/dexterity-types/Document/@@behaviors.

Reply:
A list of behaviors with behavior names and title is very important and nowhere else to find. It is super helpful for a developer to find the implementation or just add it to a type of an add-on.

Apart from this documentation chapter, I think https://6-classic.demo.plone.org/dexterity-types/Document/@@behaviors should show or offer hover tip with behavior names.
Will do a PR.

Reply:
I agree! What I think you all are missing is that I do not know where these short names can be found outside of this table. Please share the secret locations of your knowledge with everyone!

Reply:
They are in registered in ZCML and used in the FTIs of the types. To find the all names I used the browser inspect feature and looked at the values of the checkboxes of the types control panel. @ksuess improvement to show them on hover or in some other way is very valuable.

Reply:
Oh, now I understand. Let's do the following.

1. Create a new issue in the correct repo with the following content. Sorry, I don't know the correct repo, else I would do that myself.

> When browsing a specific content type from the Content Type control panel, a list of behaviors is shown in the user interface with their Titles and Description. For each behavior, one may view the source code of the checkbox (its `name` attribute) to view their Short Names (is there a better name than "Short Names"?). For example, see the content type page's behaviors: https://6-classic.demo.plone.org/dexterity-types/Document/@@behaviors
>
> ![content-type-control-panel-page-behaviors](https://user-images.githubusercontent.com/102112/208317125-a0d23006-6b4d-4ce7-bbf3-ef3ace5482b4.png)
>
> See https://github.com/plone/documentation/pull/1372#discussion_r1050794700

2. Add a `todo` below the table in this PR.

```{todo}
For each behavior in the table above, one may view the source code of the checkbox (its `name` attribute) to view its Short Name.
An issue has been created to better expose these in the user interface. [Insert link to new issue]
```

3. Sort the table items by their `Title` to align with their order in the control panel.
4. Upon release of the new feature, delete the `todo`.

Reply:
If in doubt, then file the issue under Products.CMFPlone. And since the control panel is defined there (IIRC) it is the perfect place.

Reply:
1. Issue created https://github.com/plone/Products.CMFPlone/issues/3706